hw/ip/edn/doc/registers.md

```diff
@@ -448,7 +448,7 @@ an interrupt or an alert.
 ## MAIN_SM_STATE
 Main state machine state observation register
 - Offset: `0x40`
-- Reset default: `0x185`
+- Reset default: `0xc1`
```

Comment:
All changes in this file belong to the commit where you change the state encodings / add more states.

Reply:
Thanks, you're right. I'll change this in my next push.